nav2_controller/src/nav2_controller.cpp

```diff
@@ -169,7 +169,7 @@ void ControllerServer::followPath()
 
       if (action_server_->is_cancel_requested()) {
```

Comment:
If the action was cancelled, I would think you'd terminate all goals, not just the current.

Reply:
That's correct, changing on bt_navigator, planner, and recoveries as well.